new editors post

stefaniebutland opened the pull request.

stefaniebutland commented:
Dearest @maelle, how do I make `class = "pull-left"` on an image not pull everything after it?
Please see [preview](https://deploy-preview-52--ropensci.netlify.app/blog/2020/10/27/editors2020/)

maelle commented:
Interesting! I think mostly we need more text in the bio, as seen in the section about Laura now. But in any case I'd also recommend using the imgtxt shortcode which I've done for Julia and Mauro: the alignment looks better.

maelle commented on content/blog/2020-10-27-editors2020/index.md:
```diff
@@ -0,0 +1,49 @@
+
+Laura is ... bio
+
+Laura on [GitHub](https://github.com/ldecicco-USGS), [Twitter](https://twitter.com/DeCiccoDonk)
```
I am using so much self-control to NOT suggest adding a period here. :joy_cat:

stefaniebutland replied:
You are very 💪🏼

maelle replied:
Your comment is distressing too. 😂 just kidding.

maelle commented on data/team/team.json:
```diff
@@ -222,6 +222,18 @@
+{
+   "name": "Julia Gustavsen",
+   "url": "https://www.juliagustavsen.com/",
+   "image" : "/img/team/julia-gustavsen.jpg",
```
I took this from Julia's website source.

stefaniebutland commented:
> I'd also recommend using the imgtxt shortcode which I've done for Julia and Mauro: the alignment looks better.

Yes it does! 
I started to change to imgtxt shortcode and then figured I should do pull-left to test what we wrote in blog guide. 

Thank you for those tests and edits.

maelle commented on content/author/julia-gustavsen/_index.md:
```diff
@@ -0,0 +1,7 @@
+---
+name: Julia Gustavsen
```
@stefaniebutland it will be added by my dev guide PR

maelle commented on content/blog/2020-10-27-editors2020/index.md:
```diff
@@ -0,0 +1,73 @@
+
+{{< imgtxt src = "/img/team/julia-gustavsen.jpg" width = "200" alt = "headshot of Julia Gustavsen" >}} 
+
+Julia Gustavsen is a Bioinformatician at the health-tech company [SOPHiA GENETICS](https://www.sophiagenetics.com/) in Lausanne (Switzerland). Julia honed her bioinformatics skills during her PhD in marine microbial ecology from the University of British Columbia (Canada). Julia is a co-maintainer of [rsnps](https://github.com/ropensci/rsnps) and has enjoyed reviewing packages for rOpenSci since  ...
```
```suggestion
Julia Gustavsen is a Bioinformatician at the health-tech company [SOPHiA GENETICS](https://www.sophiagenetics.com/) in Lausanne (Switzerland). Julia honed her bioinformatics skills during her PhD in marine microbial ecology from the University of British Columbia (Canada). Julia is a co-maintainer of [rsnps](https://docs.ropensci.org/rsnps) and has enjoyed reviewing packages for rOpenSci since 2015. 
```

stefaniebutland commented:
@maurolepore @ldecicco-USGS @jooolia Thank you for your bios and your generous quotes. Here's the [preview of the post](https://deploy-preview-52--ropensci.netlify.app/blog/2020/10/27/editors2020/). 

Please review by Fri Oct 23 if possible (before Mon Oct 26 is ok). No edits required, but you're welcome to request/make edits using commit suggestions or comments. 

**Items to consider**
- check each other's bios to see if you want to edit yours
- fyi Julia your rOpenSci author page will be live on Thurs Oct 22.
- Julia, the others listed packages they reviewed & edited, so you could add those if you like (not required; links below if you want to add)
  - Editor [treedata.table](https://github.com/ropensci/software-review/issues/367)
  - [googleLanguageR](https://github.com/ropensci/software-review/issues/127)
  - [plateR](https://github.com/ropensci/software-review/issues/60)
  - [rusda](https://github.com/ropensci/software-review/issues/18)
  - [FedData](https://github.com/ropensci/software-review/issues/13)

**I made these changes**
- Laura, I added links to unconf blog posts and packages reviews to your bio
- Mauro, I fixed a typo and changed tense in bio to clarify

maurolepore reviewed (approved):
LGTM,
Thanks!
